tests/Rule/LatteTemplatesRule/SimpleControl/config.neon

```diff
@@ -0,0 +1,4 @@
+parameters:
+    latte:
+        collectedPaths:
```

Comment:
You should create parent and grandparent in source folder and not collect data from whole fixtures directory

Reply:
Or run analyse on 3 files instead of one

Reply:
It does not work because analyse is called on every file seaprately in tests. And I did not wanted to move it to Source when it is just for single fixture.

It is done same way in PresenterWithoutMOdule

Reply:
Can I keep it this way or do you want me to move it to source even for Fixture specific code?

Reply:
keep it